crates/libcontainer/src/container/builder_impl.rs

```diff
@@ -38,6 +38,8 @@ pub(super) struct ContainerBuilderImpl<'a> {
     pub container: Option<Container>,
     /// File descriptos preserved/passed to the container init process.
     pub preserve_fds: i32,
+    /// Denotes if the builder is for the exec call or not
+    pub is_exec_builder: bool,
```

Comment:
How about using the `init` field instead of that?
- If `init` is true, `is_exec_builder` is false
- If `init` is false, `is_exec_builder` is true